src/jit/CMakeLists.txt

```diff
@@ -79,6 +79,15 @@ elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
     codegenarm.cpp
     unwindarm.cpp
   )
+elseif(CLR_CMAKE_PLATFORM_ARCH_I386)
+  set( ARCH_SOURCES
+    emitxarch.cpp
+    targetx86.cpp
+    lowerxarch.cpp
+    codegenxarch.cpp
+    codegenlegacy.cpp
```

Comment:
I did a test with `LEGACY_BACKEND` and forgot to remove those, I'll cleanup. Thanks!